src/components/calcite-time-picker/calcite-time-picker.tsx

```diff
@@ -0,0 +1,702 @@
+            <span
+              aria-label="increase minute"
+              class="minute-up shift"
+              onClick={this.incrementMinute}
+              onKeyDown={this.incrementMinute}
+              role="button"
+              tabIndex={-1}
+            >
+              <calcite-icon icon="chevronup" scale={iconScale} />
+            </span>
+            <span
+              aria-label="Minute"
+              aria-valuemax="12"
+              aria-valuemin="1"
+              aria-valuenow={this.minute !== "--" ? parseInt(this.minute) : undefined}
+              aria-valuetext={this.minute !== "--" ? this.minute : undefined}
+              class="minute input"
+              onFocus={this.focusHandler}
+              onKeyDown={this.minuteKeyDownHandler}
+              ref={(el) => (this.minuteEl = el)}
+              role="spinbutton"
+              tabIndex={0}
+            >
+              {this.minute}
+            </span>
+            <span
+              aria-label="decrease minute"
+              class="minute-down shift"
+              onClick={this.decrementMinute}
+              onKeyDown={this.decrementMinute}
+              role="button"
+              tabIndex={-1}
+            >
+              <calcite-icon icon="chevrondown" scale={iconScale} />
+            </span>
+          </div>
+          {includeSeconds && <span class="colon">:</span>}
+          {includeSeconds && (
+            <div>
+              <span
+                aria-label="increase second"
+                class="second-up shift"
+                onClick={this.incrementSecond}
+                onKeyDown={this.incrementSecond}
+                role="button"
+                tabIndex={-1}
+              >
+                <calcite-icon icon="chevronup" scale={iconScale} />
+              </span>
+              <span
+                aria-label="Second"
+                aria-valuemax="59"
+                aria-valuemin="0"
+                aria-valuenow={this.second !== "--" ? parseInt(this.second) : undefined}
+                aria-valuetext={this.second !== "--" ? this.second : undefined}
+                class="second input"
+                onFocus={this.focusHandler}
+                onKeyDown={this.secondKeyDownHandler}
+                ref={(el) => (this.secondEl = el)}
+                role="spinbutton"
+                tabIndex={0}
+              >
+                {this.second}
+              </span>
+              <span
+                aria-label="decrease second"
+                class="second-down shift"
+                onClick={this.decrementSecond}
+                onKeyDown={this.decrementSecond}
+                role="button"
+                tabIndex={-1}
+              >
+                <calcite-icon icon="chevrondown" scale={iconScale} />
+              </span>
+            </div>
+          )}
+          {this.hourDisplayFormat === "12" && (
+            <div>
+              <span
+                aria-label="switch to am or pm"
```

Comment:
these will need to be `intl-` props.

Reply:
What should the name be you think? `intl-meridiem`?  and `intl-minute`, `intl-second` `intl-hour`, etc..

Reply:
I think we should maybe just provide these translated as part of calcite components. Otherwise literally every team using it will have to add the strings to their translation process. We did some intl work in date, so there is precendence

Reply:
@paulcpederson does this require some lead time to get those translations back from the translations team?

Reply:
Yes. I'm not sure we've set up a true translation pipeline for CC yet, but we could work with the team and do that.

Reply:
Cool.  In the meantime I'll update these to be props with the `intl-` prefix.

Reply:
Done: https://github.com/Esri/calcite-components/commit/6fec99a4b807673c22030b8ffc2d0266fb44a0fe